Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -279,6 +279,9 @@ theorem zero_or_succ_or_limit (o : Ordinal) : o = 0 ∨ (∃ a, o = succ a) ∨
     if h : ∃ a, o = succ a then Or.inr (Or.inl h)
     else Or.inr <| Or.inr ⟨o0, fun _a => (succ_lt_of_not_succ h).2⟩
 
+theorem isLimit_of_not_succ_of_ne_zero {o : Ordinal} (h : ¬∃ a, o = succ a) (h' : o ≠ 0) :
```

Comment:
You don't need to change anything, but fyi `IsLimit` will probably be defined in terms of `IsSucc(Pre)limit` soon and this theorem will be deprecated.